### PR TITLE
Check env file path on astro deploy

### PR DESCRIPTION
**Pull request by kushalmalani**
## Description

> Describe the purpose of this pull request.

Check env file path on astro deploy

## 🎟 Issue(s)

Related #874 

## 🧪 Functional Testing

> List the functional testing steps to confirm this feature or fix.

## 📸 Screenshots

> Add screenshots to illustrate the validity of these changes.

Screen Shot 2022-11-16 at 17 32 46

## 📋 Checklist

- [x] Rebased from the main (or release if patching) branch (before testing)
- [x] Ran `make test` before taking out of draft
- [x] Ran `make lint` before taking out of draft
- [x] Added/updated applicable tests
- [ ] Tested against [Astro-API](https://github.com/astronomer/astro/) (if necessary).
- [ ] Tested against [Houston-API](https://github.com/astronomer/houston-api/) and [Astronomer](https://github.com/astronomer/astronomer/) (if necessary).
- [ ] Communicated to/tagged owners of respective clients potentially impacted by these changes.
- [ ] Updated any related [documentation](https://github.com/astronomer/docs/)

### CODE REVIEW

File: cmd/cloud/organization_test.go [hunk 1]
```diff
@@ -73,4 +74,14 @@ func TestOrganizationExportAuditLogs(t *testing.T) {
 		_, err := execOrganizationCmd(cmdArgs...)
 		assert.NoError(t, err)
 	})
+
+	// Delete audit logs exports
+	currentDir, _ := os.Getwd()
```
Comment (kushalmalani) [inline cmd/cloud/organization_test.go:79]: these test cases where leaving exported files. just patching those test cases to delete those exported files

File: cloud/deploy/deploy.go [hunk 2]
```diff
@@ -248,6 +251,11 @@ func Deploy(deployInput InputDeploy, client astro.Client) error { //nolint
 			fmt.Sprintf("\n Deployment View: %s", ansi.Bold(deploymentURL)) +
 			fmt.Sprintf("\n Airflow UI: %s", ansi.Bold(deployInfo.webserverURL)))
 	} else {
+		envFileExists, _ := fileutil.Exists(deployInput.EnvFile, nil)
+		if !envFileExists {
+			return envFileMissing
```
Comment (jemishp) [inline cloud/deploy/deploy.go:256]: nit: consider wrapping the error with the `deployInput.EnvFile` for a better UX. For e.g. below:

`return fmt.Errorf("%w %s", envFileMissing, deployInput.EnvFile)`

File: cmd/cloud/organization_test.go (hunk 1 above)
Comment (jemishp) [reply cmd/cloud/organization_test.go:79]: yes I noticed them too. I was wondering why they were showing up when `execOrganizationCmd` already sets its output to go to a buffer as opposed to `os.stdout`

**Review by jemishp — APPROVED** [review]
LGTM

**Pull request merged**